code/modules/surgery/organs/augments_internal.dm

```diff
@@ -286,6 +286,95 @@
 	origin_tech = "materials=6;programming=6;biotech=6"
 	emp_proof = TRUE
 
+// An implant that injects you with twitch on demand, acting like a bootleg sandevistan
+
+/obj/item/organ/internal/cyberimp/brain/sensory_enhancer
+	name = "\improper Qani-Laaca sensory computer"
+	desc = "An experimental implant replacing the spine of organics. When activated, it can give a temporary boost to mental processing speed, \
+		Which many users percieve as a slowing of time and quickening of their ability to act. Due to its nature, it is incompatible with \
+		system that heavily influence the user's nervous system, like the central nervous system rebooter. \
+		As a bonus effect, you are immune to the burst of heart damage that comes at the end of twitch usage, as the computer is able to regulate \
+		your heart's rhythm back to normal after its use."
+	icon_state = "sandy"
+	implant_overlay = null
+	implant_color = null
+	slot = "brain_antistun"
+	actions_types = list(/datum/action/item_action/organ_action/toggle/sensory_enhancer)
+	COOLDOWN_DECLARE(sensory_enhancer_cooldown)
+
+
+/obj/item/organ/internal/cyberimp/brain/sensory_enhancer/insert(mob/living/carbon/M, special = 0)
+	. = ..()
+	ADD_TRAIT(M, TRAIT_TWITCH_ADAPTED, "[UID()]")
+
+/obj/item/organ/internal/cyberimp/brain/sensory_enhancer/remove(mob/living/carbon/M, special = 0)
+	. = ..()
+	REMOVE_TRAIT(M, TRAIT_TWITCH_ADAPTED, "[UID()]")
+
+/obj/item/organ/internal/cyberimp/brain/sensory_enhancer/render()
+	var/mutable_appearance/our_MA = mutable_appearance('icons/mob/human_races/robotic.dmi', icon_state, layer = -INTORGAN_LAYER)
+	return our_MA
+
+/datum/action/item_action/organ_action/toggle/sensory_enhancer
+	name = "Activate Qani-Laaca System"
+	desc = "Activates your Qani-Laaca computer and grants you its powers. LMB: Short, safer activation. ALT/MIDDLE: Longer, more powerful, more dangerous activation."
```

Comment:
We usually use those terms (LMB/ALT/MIDDLE) in other places?

Reply:
LMB is used on modsuits, MMB is used on modsuits, I could do LMB ALT MMB instead

Reply:
Do whatever you think should be the standard for these buttons